pkg/networkservice/common/mechanismtranslation/client_test.go

```diff
@@ -38,6 +38,9 @@ import (
 
 func kernelMechanism() *networkservice.Mechanism {
 	request := new(networkservice.NetworkServiceRequest)
```

Comment:
Probably this would be better?
```suggestion
	request := &networkservice.NetworkServiceRequest{
		Connection: &networkservice.Connection{
			Id: "id",
		},
	},
```

Reply:
Fixed